### DISCUSSION TITLE
Display png image where desktop wallpaper shows through image transparency

### DISCUSSION THREAD

**Discussion by ke-ss** [ImageMagick/ImageMagick#5885] [General]
I'm running latest version of Openbox & Arch linux. 

How can I display a png image with background transparency, without any surrounding border or frame, at the bottom centre of the screen so that the desktop wallpaper shows through the image transparency?

Did quite a few searches and came up with nothing.

If ImageMagick won't do it is there a tool that will.

**Comment by snibgo** [comment]
ImageMagick can't do that.

In Windows, viewing programs can be written where the only visible components are opaque or partially-transparent pixels in the displayed image. I guess the same is possible on Linux, and probably in GUI toolkits.
